src/api/app/models/user.rb

```diff
@@ -106,6 +106,11 @@ def track_create
     RabbitmqBus.send_to_bus('metrics', 'user.create value=1')
   end
 
+  def roles_for_metrics
+    # We add all the roles because otherwise false won't be equivalent to not true
+    Roles.all.map { |role| "#{role.title}=#{roles.exists?(role.id)}" }.join(',')
```

Comment:
This would would run for _every_ request. Not such a good idea. We should rather add the global roles to the session or something...